templates/glue-tables.j2

```diff
@@ -0,0 +1,60 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: The set of Glue tables created
+
+Parameters:
+  Namespace:
+    Type: String
+    Description: >-
+      The namespace string used to build up the path to the correct object keys
+      in the bucket
+    Default: recover
+
+# buckets
+  S3IntermediateBucketName:
+    Type: String
+    Description: Name of the S3 intermediate (JSON) bucket
+
+Resources:
+  # combine all configuration data into a "datasets" collection
+  {% set datasets = [] %}
+  {% for v in sceptre_user_data.dataset_schemas.tables.keys() %}
+  {% set dataset = {} %}
+  {% do dataset.update({'dataset_name': v}) %}
```

Comment:
I'm looking at the template in BridgeDownstream, can you explain why these [lines](https://github.com/Sage-Bionetworks/BridgeDownstream/blob/8994ffde9240175d36488f5ccb8cba8e5667390e/templates/study-pipeline-infra.j2#L102-L106) aren't needed? 

Is it because there are no crawlers?

Reply:
Yes currently, we don't have any plans for glue crawlers or partitions in the glue tables.